src/update-client.cc

```diff
@@ -339,9 +341,25 @@ void FileUpdater::update()
 			fs::rename(to_path, old_file_path);
 
 		fs::rename(from_path, to_path);
+		
+		reset_rights(to_path);
 	}
 }
 
+bool FileUpdater::reset_rights(const fs::path& path)
+{
+	ACL empty_acl;
+	if (InitializeAcl(&empty_acl, sizeof(empty_acl), ACL_REVISION)) 
+	{
+		DWORD result = SetNamedSecurityInfo((LPWSTR)path.c_str() ,SE_FILE_OBJECT, DACL_SECURITY_INFORMATION | UNPROTECTED_DACL_SECURITY_INFORMATION, 0, 0, &empty_acl, 0  );
```

Comment:
Just a weird spacing here, can you quickly fix that? Order than that, the flags look good

Reply:
done. 
I have apply auto code format from visual studio on that new function.

Reply:
Perfect, thanks 👍 Merging